backend/src/main/java/com/team5/issue_tracker/issue/dto/request/IssueSearchRequest.java

```diff
@@ -17,4 +17,38 @@ public class IssueSearchRequest {
   private Set<String> labelNames;
   private String milestoneName;
   private String authorName;
+
+  public String toQueryString() {
+    StringBuilder queryString = new StringBuilder();
+    if (isOpen != null) {
+      queryString.append("is:").append(isOpen ? "open" : "closed").append(" ");
+    }
+
```

Comment:
append 하는 것이 모두 같은 패턴이라 유틸성 메서드를 만들어두면 가독성이 향상 될 것 같아요. 제안에 대해 한번 고민해보세요

```suggestion
  public String toQueryString() {
    StringBuilder queryString = new StringBuilder();

    appendCondition(queryString, "is", isOpen == null ? null : (isOpen ? "open" : "closed"));
    appendCondition(queryString, "assignee", assigneeName);
    // TODO 코드 완성

    return queryString.toString().trim();
  }

  private void appendCondition(StringBuilder sb, String key, String value) {
    if (hasText(value)) {     sb.append(key).append(":").append(quoteIfContainsWhitespace(value)).append(" ");
    }
  }
```